region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -1095,10 +1095,30 @@
         "SpaceJump"
       ]
     },
+    {
+      "link": [5, 8],
+      "name": "Tricky Spring Ball Jump",
+      "requires": [
+        "canTrickySpringBallJump",
+        "canTrickyJump"
+      ],
+      "note": [
+        "Run and spin jump into a spring ball jump, starting from the floating platform."
+      ]
+    },
+    {
+      "link": [5, 8],
+      "name": "Tricky Dash Jump",
+      "requires": [
+        "canTrickyDashJump",
+        "Morph"
+      ]
```

Comment:
Could use a bit of a description - looks like it doesnt use the full runway length. Looks like it's missing HiJump as well.